e2e/_suites/kubernetes-autodiscover/kubernetes_test.go

```diff
@@ -0,0 +1,174 @@
+package main
```

Comment:
I think you can remove the `_test` from this file, as it's not a test by itself but a library used in tests. Does it make sense from Go native standpoint?

Reply:
I actually added `_test` to both files because I thought it was the convenience with godog :slightly_smiling_face: In principle code that is only used by tests can live in `_test` files, but no strong opinion here. Suffix removed.